nilearn/conftest.py

```diff
@@ -484,6 +484,33 @@ def surf_label_img(surf_mesh):
     return SurfaceImage(surf_mesh(), data)
 
 
+@pytest.fixture
+def surf_maps_img(surf_mesh):
+    """Return a sample surface map image using the sample mesh.
+    Has 4 different overlapping regions per hemisphere.
```

Comment:
Wouldn't it be better to have different numbers of regions per hemisphere ?

Reply:
Hmm... I agree that's a possibility in maps images, but I don't think we allow having different "n_timepoints" across hemispheres in SurfaceImage as of now. See:

https://github.com/nilearn/nilearn/blob/5271b3234ba450205060349b7cd401044d099220/nilearn/surface/surface.py#L1219

Reply:
On that note, I don't think we have a probabilistic atlas for surfaces. I wanted to write an example to test out the reports...